Tests/AccountTests/Push/AutopushTests.swift

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+import MozillaAppServices
+import Account
+
+class AutopushTests: XCTestCase {
+    private var mockPushManager = MockPushManager()
+    private var autopushClient: Autopush?
+
+    override func setUp() async throws {
+        autopushClient = Autopush(withPushManager: mockPushManager)
+        try await super.setUp()
+    }
+
+    override func tearDown() {
+        autopushClient = nil
+        mockPushManager = MockPushManager()
```

Comment:
We normally `nil` variables in tearDown to ensure they don't live past the test lifecycle. Every tests create individual instance and doesn't deinit it after completion. With a tearDown we ensure we properly clear any instances and start with a clean slate on each test case.
```suggestion
        mockPushManager = nil
```